templates/systemd/funkwhale-frontend.service.j2

```diff
@@ -29,6 +29,7 @@ ExecStartPre={{ devture_systemd_docker_base_host_command_docker }} create \
                 --mount type=bind,src={{ funkwhale_media_path }},dst=/srv/funkwhale/data/media,readonly \
                 --mount type=bind,src={{ funkwhale_static_path }},dst=/usr/share/nginx/html/staticfiles,readonly \
                 --env-file={{ funkwhale_frontend_env_path }} \
+                {{ funkwhale_frontend_process_extra_arguments | join(' ') }} \
```

Comment:
This should probably be `funkwhale_frontend_container_extra_arguments`, as the arguments are passed to `docker run`, not to the process.